README.md

```diff
@@ -85,6 +85,12 @@ ruby bundle
 - [Twitter](https://twitter.com/MohamedCK0)
 - [LinkedIn](https://www.linkedin.com/in/mohamed-abdulhakim-2868521b6/)
 
+👤 Harrison Njuguna
+
+- GitHub: https://github.com/Harrynoqs)
+- Twitter: https://twitter.com/noqsnoq1
+- LinkedIn:https://linkedin.com/in/harrisonw-njuguna
+
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
```

Comment:
- [OPTIONAL] Please kindly update your acknowledgement section to reflect the changes for this milestone, as this will make your readme file more informative and professional. I think `Nelson Sakwa` has nothing to do with this project, please kindly remove that.

Reply:
Thanks for the reminder, It's done. 😄